packets/game_state_packet.cpp

```diff
@@ -19,46 +19,52 @@ size_t GameStatePacket::calculateSize() const {
 // Ensure outData is large enough to store all the data that is intended to be serialized.
 void GameStatePacket::serialize(const GameStatePacket& packet, char*& outData) {
     char* temp = outData;
+    int size = 0;
```

Comment:
what is the use of size? I think it could be removed to clean up the code

Reply:
Oh oops that was a debug variable I forgot to remove

Reply:
removed